server/spec/test_data_provider_canvas_esb.rb

```diff
@@ -102,4 +102,15 @@ def test_upcoming_events_studenta
     assert_operator title_0.length, '>', 0, "title has length"
   end
 
+  #### test that the array parameters are expanded correctly.
+  #self.urlArrayParameters('&state[]=',allowed_states)
+  #self.urlArrayParameters
+  #DataProviderCanvasESB
+
+  def test_urlArrayParameters
+    r = DataProviderCanvasESB.urlArrayParameters("&prefix[]=", ["B", "C"])
```

Comment:
They don't stand for anything.  They are just strings for testing.  There is no need to be random.  They can just be arbitrary.  This will also go away when I remove the currently unused urlArrayParameters method.